config_defaults/subtests/docker_cli/negativeusage.ini

```diff
@@ -70,6 +70,24 @@ subarg = -e,PATH=/tmp,%%(FQIN)s,true
 stderr = exec\: \"true\"\: executable file not found in \$PATH
 extcmd = 127
 
+[docker_cli/negativeusage/iv5]
+subcmd = run
+subarg = --tmpfs,/dev,%%(FQIN)s
+stderr =  no such file or directory
+extcmd = 127
+
+[docker_cli/negativeusage/iv6]
+subcmd = run
+subarg = --tmpfs,:rw,%%(FQIN)s
+stderr =  open /proc/self/task/1/attr/exec: no such file or directory
+extcmd = 125
+
+[docker_cli/negativeusage/iv7]
+subcmd = run
+subarg = --tmpfs,.,%%(FQIN)s
+stderr =  Message: Failed to open /dev/null - open /dev/null: permission denied
+extcmd = 125
+
```

Comment:
@cevich with subarg = --tmpfs,,%%(FQIN)s, "/usr/bin/docker-latest -D run --tmpfs  registry.access.redhat.com/rhel7/rhel:latest" is executed and gives 'docker run --invalid-opt invalid-image'